third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -3576,6 +3622,7 @@ func flattenMaintenancePolicy(mp *containerBeta.MaintenancePolicy) []map[string]
 						"duration":   mp.Window.DailyMaintenanceWindow.Duration,
 					},
 				},
+				"maintenance_exclusion": exclusionWindows,
```

Comment:
Does maintenance exclusion apply to both daily and recurring windows? Can you one of both of the types of windows to the test?

Reply:
done